scripts/backtest_sites.py

```diff
@@ -327,18 +326,18 @@ def predict_batch(self, batch: NumpyBatch) -> xr.Dataset:
             y_normed_site = model(device_batch).detach().cpu().numpy()
         da_normed_site = preds_to_dataarray(y_normed_site, model, valid_times, ALL_SITE_IDS)
 
-        # TODO fix this step: Multiply normalised forecasts by capacities and clip negatives
-        # For now output normalised by capacity outputs and unnormalise in post processing
-        # da_abs_site = da_normed_site.clip(0, None) * site_capacities[:, None, None]
-        da_normed_site = da_normed_site.clip(0, None)
+        # Multiply normalised forecasts by capacities and clip negatives
+        da_abs_site = da_normed_site.clip(0, None) * site_capacities[:, None, None]
+        da_abs_site = da_abs_site.clip(0, None)
```

Comment:
I think you don't need it with the previous clip added back in?

Reply:
I like that you renamed it btw, that's much better

Reply:
Yep good spot, remove that duplicate now, thanks